packages/subgraph/schema.graphql

```diff
@@ -1,181 +1,460 @@
+  id: ID!
+
+  " The DAO that holds the Tokens"
   dao: Dao!
+
+  " The block timestamp of the last update (transfer) for this token balance. "
   lastUpdated: BigInt!
 }
 
+"""
+The ERC20 Token balances for the DAO.
+"""
 type ERC20Balance implements TokenBalance @entity {
+  " The concatenation of DAO Smart contract address and Token Smart contract address. Joined by '_'. "
   id: ID!
+
+  " The [ERC20 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-20) instance. "
   token: ERC20Contract!
+
+  " The DAO that holds the ERC20 token. "
   dao: Dao!
+
+  " The amount of ERC20 tokens held by the DAO. "
   balance: BigInt!
+
+  " The block timestamp of the last update (transfer) for this token balance. "
   lastUpdated: BigInt!
 }
 
+"""
+The Native token balances of the DAO, representing the amount of the blockchain's native currency
+(e.g., Ether on Ethereum, MATIC on Polygon) held by the DAO.
+"""
 type NativeBalance implements TokenBalance @entity {
+  " The concatenation of DAO Smart contract address and Token Smart contract address. Joined by '_'. "
   id: ID!
+
+  " The DAO that holds the native tokens"
   dao: Dao!
+
+  " The amount of native tokens held by the DAO. "
   balance: BigInt!
+
+  " The block timestamp of the last update (transfer) for this native token balance. "
   lastUpdated: BigInt!
 }
 
+"""
+The NFT balances of the DAO. Including all Token IDs held by the DAO per [ERC721 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-721).
+"""
 type ERC721Balance implements TokenBalance @entity {
+  " The concatenation of DAO Smart Contract address and [ERC721 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-721) address. Joined by '_'. "
   id: ID!
+
+  " The [ERC721 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-721) instance. "
   token: ERC721Contract!
+
+  " The DAO that holds tokens from a [ERC721 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-721). "
   dao: Dao!
+
+  " The NFTs from one ERC721 Contract held by the DAO. "
   tokenIds: [BigInt!]!
+
+  " The block timestamp of the last update (transfer) for this token balance. "
   lastUpdated: BigInt!
 }
 
+"""
+The ERC1155 Token balances of the DAO. Including all Token IDs and it's amounts held by the DAO per [ERC1155 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-1155).
+"""
 type ERC1155Balance implements TokenBalance @entity {
+  " The concatenation of DAO Smart Contract address and [ERC1155 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-1155) address. Joined by '_'. "
   id: ID!
+
+  " The [ERC1155 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-1155) instance. "
   token: ERC1155Contract!
+
+  " The DAO that holds tokens from a [ERC1155 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-1155). "
   dao: Dao!
+
+  " The amount and related information of the DAO held tokens per [ERC1155 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-1155). "
```

Comment:
this could be worded better

Reply:
Updates as:
Balances for each token ID from a [ERC1155 Token Smart Contract](https://eips.ethereum.org/EIPS/eip-1155).